src/libraries/pkg/baseline/packageIndex.json

```diff
@@ -772,6 +772,56 @@
         "6.0.0.0": "6.0.0"
       }
     },
+    "Microsoft.Extensions.Hosting.Systemd": {
+      "StableVersions": [
+        "3.0.0",
+        "3.0.1",
+        "3.0.2",
+        "3.0.3",
+        "3.1.0",
+        "3.1.1",
+        "3.1.2",
+        "3.1.3",
+        "3.1.4",
+        "3.1.5",
+        "3.1.6",
+        "3.1.7",
+        "3.1.8",
+        "3.1.9",
+        "3.1.10",
+        "5.0.0"
```

Comment:
It looks like there are `5.0.1` versions of both of these packages on nuget.org.

[https://www.nuget.org/packages/Microsoft.Extensions.Hosting.WindowsServices](https://www.nuget.org/packages/Microsoft.Extensions.Hosting.WindowsServices)